zuul.d/cisco-iosxr-jobs.yaml

```diff
@@ -223,6 +223,20 @@
       ansible_test_python: 3.6
       ansible_test_split_in: 2
 
+- job:
+    name: ansible-test-network-integration-iosxr-network_cli-libssh-python36-scenario01
+    parent: ansible-test-network-integration-iosxr-network_cli-python36
```

Comment:
lets parent to
```
ansible-test-network-integration-iosxr-network_cli-python36-scenario01
```
Then you only need to set
```
ansible_test_network_cli_ssh_type: libssh
```

Reply:
ack